Sources/WebSocket.swift

```diff
@@ -479,7 +479,7 @@ open class WebSocket : NSObject, StreamDelegate, WebSocketClient, WSStreamDelega
      - parameter completion: The (optional) completion handler.
      */
     open func write(string: String, completion: (() -> ())? = nil) {
-        guard isConnected else { return }
+        guard !isConnected else { return }
```

Comment:
This shouldn't change right?

Reply:
This is not the root of the problem. It's 

 public func write(data: Data) -> Int {
        guard let outStream = outputStream else {return 0}
        let buffer = UnsafeRawPointer((data as NSData).bytes).assumingMemoryBound(to: UInt8.self)
        return outStream.write(buffer, maxLength: data.count)
    }

Problem